### DISCUSSION TITLE
Your idea blew my mind!

### DISCUSSION THREAD

**Discussion by agnoldo** [DeabLabs/cannoli#2] [General]
Dear developer(s),

I just wanted to tell you that your idea blew my mind! Congratulations, this is definitely thinking out of the box, even more if we consider that part of the thinking is done by OpenAI...

Amazing! I will test it a lot. Can't wait to explore the possibilities...

Thank you very much!

Arnaldo

**Comment by blindmansion** [comment]
Thank you for trying Cannoli! Great to hear it's sparked some ideas, would love to hear what you build. Please let us know if you encounter any bugs or have ideas for features.
